packages/sdk/package.json

```diff
@@ -9,8 +9,9 @@
   ],
   "scripts": {
     "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint",
+    "prebuild": "npx ts-node scripts/generateTokenList.ts",
```

Comment:
The indexer in future will use the standard token list to find custom bridges it needs to index